aiidalab_eln/cheminfo/exporter.py

```diff
@@ -1,5 +1,4 @@
 from pytojcamp import from_dict
-from cheminfopy import Sample
 
 
 def export_isotherm(
```

Comment:
I can update the export isotherm function in an hour or so. 
We should probably also provide temperature, maybe the force field / settings.

Reply:
Ok, then let's not worry about this particular function here right now. As I've mentioned earlier - we have no isotherm data object in AiiDA just yet. Once it is created - we can reaccess it.